install/macos_packages/readme.md

```diff
@@ -0,0 +1,262 @@
+# MacOS Codesigning Scripts
```

Comment:
Absolutely *love* this documentation! 😍 
Hopefully we won't need to refer to it too often 🤞